examples/rpl-udp/udp-client.c

```diff
@@ -59,7 +59,8 @@ PROCESS_THREAD(udp_client_process, ev, data)
       LOG_INFO("Sending request %u to ", count);
       LOG_INFO_6ADDR(&dest_ipaddr);
       LOG_INFO_("\n");
-      snprintf(str, sizeof(str), "hello %d", count);
+      /* avoid the risk of not fitting the '\0' in the string */
+      snprintf(str, sizeof(str) - 1, "hello %d", count);
```

Comment:
I believe `snprintf` *always* writes the null terminating char

Reply:
true - will fix. There might be some old Microsoft snprintf that did not.

Reply:
please give a heads up when fixed, then I'm ready to approve this PR!

Reply:
this is fixed in the last commit.